.scripts/set-version.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
It'd be nice if we didn't add this so that it's formatted like the rest of the project. But that can be a future improvement.